src/otx/engine/engine.py

```diff
@@ -161,6 +161,8 @@ def train(
         metric: Metric | MetricCallable | None = None,
         run_hpo: bool = False,
         hpo_config: HpoConfig | None = None,
+        warmup_steps: int = 0,
+        warmup_by_epochs: bool = False,
```

Comment:
I'm still not sure if `warmup_by_epochs` is necessary.
Anyway, could you please add a docstring for this argument?

Reply:
Done, https://github.com/openvinotoolkit/training_extensions/pull/3035/commits/fceca297d1d4bf7e892a6f75f3da4fcfde943caa